docs/essential/04_dummy_classes.md

```diff
@@ -164,9 +164,18 @@ struct RBCollisionChannelContainer
     var() const bool Clothing;
     var() const bool ClothingCollision;
 };
-
+```
 This allows us to give objects different collision channels for different behaviors. VehicleBlocker objects allow the ball through, but not the player. BallBlocker is the reverse. And Ball is… like the ball.
 
+On line ~180, update 
+```unrealscript{10-12}
```

Comment:
Numbers between brackets are for line highlighting and since there is no line 10-12 it can be removed (to ```unrealscript). Same for line 175

Reply:
Good eye, you caught me on my copy pasta. Fixed!